config/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Yeah, mine is also failing due to trailing whitespace. I'm on a windows system right now though, with Gradle 2.2.1, here's the exact error message I'm getting:
http://pastebin.com/ShLbHa00
Mostly it's just complaining about whitespace in the previous code. Does Gradle possess and easy way to autoformat things?